doc/man3/SSL_CTX_set0_CA_list.pod

```diff
@@ -42,47 +71,124 @@ SSL_add1_to_CA_list() appends the CA subject name extracted from B<x> to the
 list of CAs sent to the peer for B<s>, overriding the setting in the parent
 B<SSL_CTX>.
 
-SSL_get0_peer_CA_list() retrieves the list of CA names (if any) the peer
-has sent.
+The following functions are very similar to the functions described above except
+that they only have an effect when called on the server side. The list of CA
+names managed are separate - so you cannot (for example) set CA names using the
+functions above and then get them using the functions below. Where a mix of the
+above functions and the functions below have been used on the server side the
+settings set by the following functions take precedence. These functions are
```

Comment:
Here, instead of "the above" and the "below" I would suggest some more direct binding, such as "dual-purpose" and "client-specific", or similar.  Indeed perhaps name to the two classes of functions when first introducing them, so that the category names are already "established" by the time we get to this point.